.github/workflows/commit_check.yaml

```
@@ -3,65 +1,167 @@
+      
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ needs.setup.outputs.python-version }}
+          cache: 'pip'
+          
+      - name: Restore virtual environment
+        uses: actions/cache@v4
+        with:
+          path: venv
+          key: venv-${{ runner.os }}-${{ env.PYTHON_VERSION }}-${{ hashFiles('requirements/dev.txt') }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)